SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -203,11 +203,19 @@ void USpatialReceiver::HandleActorAuthority(Worker_AuthorityChangeOp& Op)
 					{
 						Actor->RemoteRole = ROLE_SimulatedProxy;
 					}
+
+					Actor->OnAuthorityGained();
+				}
+				else if (Op.authority == WORKER_AUTHORITY_AUTHORITY_LOSS_IMMINENT)
```

Comment:
Perhaps this should be a switch statement now.

Reply:
Will there ever be more than the three callbacks? I think this is probably fine

Reply:
I think this is fine as well.

Reply:
This day will live in infamy.